galeracluster/source/documentation/galera-parameters.rst

```diff
@@ -1701,6 +1734,11 @@ The period for which the PC protocol waits for the EVS termination.
    "Dynamic", "No"
    "Initial Version", "1.0"
 
+When a node leaves the cluster, it will wait for ``pc.linger`` time to
+get itself removed gracefully from the cluster. If the graceful leave
+is not possible for some reason (e.g. cluster partitioning at the same

```

Comment:
I find this description a little confusing. But it's a little bit hard to explain why...
I'm not sure how this pc.linger works, but from the description I feel like something is missing.
First, statement basically says "wait for pc.linger amount of time to allow for graceful removal from".
Second statement says "if graceful shutdown is not possible, then wait for pc.linger time before terminating".
So the impression I have is that overall the behavior is always "wait for pc.linger time before terminating".
Perhaps, the node that is leaving either waits for pc.linger time, but it can detect if it left the cluster gracefully, in which case it can terminate before the timeout?